components/caption_images/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN pip3 install --no-cache-dir -r requirements.txt
 # Install Fondant
 # This is split from other requirements to leverage caching
 ARG FONDANT_VERSION=main
-RUN pip3 install git+https://github.com/ml6team/fondant@${FONDANT_VERSION}
+RUN pip3 install git+https://github.com/ml6team/fondant@${FONDANT_VERSION}#egg=fondant[aws, azure, gcp]
```

Comment:
Thanks @PhilippeMoussalli 

In the pip documentation ([example 7])(https://pip.pypa.io/en/stable/cli/pip_install/#examples), they use the following format:
```
pip install fondant[aws,azure,gcp] @ git+https://github.com/ml6team/fondant@${FONDANT_VERSION}
```

Which I think is a bit more readable and more future-proof, as eggs have been deprecated as far as I'm aware.

Reply:
oh good to know. I was not aware. I'll change them